src/GO/Job.php

```diff
@@ -498,7 +500,7 @@ private function finalise()
     /**
      * Email the output of the job, if any.
      *
-     * @return void
+     * @return bool
```

Comment:
I am sorry you are right, there is a `return false` just before so your `return true` is absolutely correct, along with the `@return bool`. My apologies can you please add it back?